static/js/browser2/in_arcs.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import axios from "axios";
+import { InArcsGroup } from "./in_arcs_group";
+
+const IGNORED_PARENT_TYPES = new Set(["StatisticalPopulation"]);
+
+interface InArcsSectionsPropType {
+  nodeName: string;
+  dcid: string;
+  inArcLabels: Array<string>;
+  provDomain: { [key: string]: URL };
+}
+interface InArcsSectionStateType {
+  parentTypes: string[];
+  arcsTableData: { [parentType: string]: { [property: string]: Array<any> } };
+}
+
+export class InArcsSection extends React.Component<
```

Comment:
try to keep the name of the file matched with the name of the name of the component. i also found it slightly confusing that "section" encapsulates multiple "groups".

Reply:
maybe I can have the section encapsulate multiple "subsection"s instead?